libs/platform/src/lib/package.json

```diff
@@ -19,12 +19,13 @@
     "@angular/animations": "ANGULAR_VER_PLACEHOLDER",
     "@angular/router": "ANGULAR_VER_PLACEHOLDER",
     "@angular/cdk": "ANGULAR_VER_PLACEHOLDER",
-    "rxjs": "RXJS_VER_PLACEHOLDER"
+    "rxjs": "RXJS_VER_PLACEHOLDER",
+    "fundamental-styles": "FDSTYLES_VER_PLACEHOLDER",
```

Comment:
do we need the FD prefix?

Reply:
Do not think we need, otherwise prepare executor should be updated as well to properly handle placeholder @platon-rov

Reply:
What do you mean? I didn't change anything except moving from deps to peerDeps.